### PR TITLE
fix: automatically trigger clear_local when permissions changed

### PR DESCRIPTION
**Pull request by sleidig**
see #7

### CODE REVIEW

File: src/admin/admin.controller.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { Controller, Param, Post, UseGuards } from '@nestjs/common';
+import { CombinedAuthGuard } from '../auth/guards/combined-auth/combined-auth.guard';
+import { OnlyAuthenticated } from '../auth/only-authenticated.decorator';
+import { AdminService } from './admin.service';
+
+/**
+ * This service provides some general administrativ endpoints.
```
Comment (sleidig) [inline src/admin/admin.controller.ts:7]:
```suggestion
 * This controller provides some general administrative endpoints.
```

File: src/admin/admin.service.ts [hunk 2]
```diff
@@ -1,17 +1,13 @@
...
+import { AllDocsResponse } from '../restricted-endpoints/replication/bulk-document/couchdb-dtos/all-docs.dto';
+import { CouchdbService } from '../couchdb/couchdb.service';
 
 /**
- * This service provides some general administrativ endpoints.
+ * Service providing some general functionalities that are required in the context of administering the db and permissions.
  */
-@OnlyAuthenticated()
-@UseGuards(CombinedAuthGuard)
-@Controller('admin')
```
[the remark this answers is not in the document]
Comment (sleidig) [reply src/admin/admin.service.ts:13]: github is interpreting this change weirdly - this is a new Service class (extracting the logic), the controller remains with the annotations (but is shown as a new file in GitHub)

[the remark this answers is not in the document]
Comment (tomwwinter) [reply src/admin/admin.service.ts:13]: Ah, i see. Alright 👍

**Review by tomwwinter — APPROVED** [review]
LGTM!

**Pull request merged**